.github/CODEOWNERS

```diff
@@ -0,0 +1,2 @@
+* @vtex-apps/store-framework-devs
+docs/ @vtex-apps/technical-writers
```

Comment:
Fix it if is it possible

Reply:
Who would you suggest as owner? Only the search team?